Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonCustomPaletteBase.cs

```diff
@@ -128,7 +128,7 @@ public KryptonCustomPaletteBase()
         /// Initialize a new instance of the KryptonPalette class.
         /// </summary>
         /// <param name="container">Container that owns the component.</param>
-        public KryptonCustomPaletteBase([DisallowNull] IContainer container)
+        public KryptonCustomPaletteBase(IContainer container)
```

Comment:
Put back the `DisallowNull`, so that the compiler can inform of useage rather the than the Debg.assert

Reply:
Reverted commit